.github/workflows/make-release.yml

```
@@ -0,0 +1,48 @@
+name:  Image release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build_and_push_to_ghcr:
+    name: Create docker release and publish to ghcr
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.20']
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: ${{ matrix.go }}
```

```suggestion
    steps:
    - name: Set up Go
      uses: actions/setup-go@v4
      with:
        go-version-file: ./go.mod
```
When releasing, better use the Go version specified in `go.mod`